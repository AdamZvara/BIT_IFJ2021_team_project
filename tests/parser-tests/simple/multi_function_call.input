require "ifj21" --toto je hlavicka 

function bar()
	write("funkce bar\n")
end

function foo(num : number)
	write(num)
	write(
	"\n"
		 )
	bar()
end

foo(5.2)
bar()
