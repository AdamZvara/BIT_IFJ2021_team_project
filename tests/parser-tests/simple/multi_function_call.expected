0x1.4cccccccccccdp+2
funkce bar
funkce bar
